microsoft-edge/webview2/concepts/power-user-faq.md

```diff
@@ -0,0 +1,33 @@
+---
+title: Power User FAQ
+description: Answers to some frequently asked questions about WebView2 from non-developers who want to learn more about it.
+author: MSEdgeTeam
+ms.author: msedgedevrel
+ms.topic: conceptual
+ms.prod: microsoft-edge
+ms.technology: webview
+ms.date: 03/29/2023
+---
+# WebView2 Power User FAQ

```

Comment:
```suggestion
# Using WebView2 apps
```
I'd avoid framing as a "FAQ" and use standard topical framing (title, doc design) instead.  Design the title along with Sentence 1 that clarifies the meaning of the title.
alts:
Using WebView2 apps
End-users info about WebView2
Information about WebView2 for end users
WebView2 Power User FAQ
WebView2 End User FAQ

whichever title is selected, propagate to the usual locations:
* title field at top
* h1 title
* TOC.yml
* link text of any links in other pages that link to here

Reply:
likely to keep as faq, per offline